### PR TITLE
Some more last styling fixes

### PR DESCRIPTION
**Pull request by Markvanderuit**
- Most windows now fixed at max 800px width, because negative space.
- Fixed some places where color was missing some times, like text fields.
- Fixed toolbar/drawer size issue. Toolbar height is 64px.
- Toolbar in coupling view now has the bloody button again.

### CODE REVIEW

File: elements/director-view/director-element.html [hunk 1]
```diff
@@ -25,6 +27,11 @@
                 position: relative;
                 margin: 0 24px;
             }
+
+            scrolling-box {
+                max-width: 400px;
+
```
Comment (mennoo1996) [inline elements/director-view/director-element.html]: Remove whitespace

File: elements/user-view/new-user-dialog.html [hunk 2]
```diff
@@ -25,11 +25,14 @@
             }
 
             paper-dialog {
-                overflow: hidden;
+                /*min-width: 400px;*/
```
Comment (mennoo1996) [inline elements/user-view/new-user-dialog.html]: Is this commented code still needed?

**Comment by martijnjanssen** [comment]
Looks good!

**Comment by mennoo1996** [comment]
Nice!

**Pull request merged**